setup.cfg

```diff
@@ -1,5 +1,9 @@
 [pytest]
-addopts = -v --cov zdict --cov-report html --cov-report term
+flake8-ignore =
+    zdict/__init__.py ALL
+    zdict/dictionaries/template.py ALL
+
+addopts = --flake8 -v --cov zdict --cov-report html --cov-report term
```

Comment:
Originally, i want to distingish the type of build failure, "Is the unittesting failure or my bad style?"
but i just found that the error message of `pytest-flake8` is quite clear to figure out that.
So maybe there is no need for the extra build.